src/components/AppSection/style.scss

```diff
@@ -0,0 +1,64 @@
+body{
+    background-color: #F5F9FD;
+}
+
+.ApplicationSection{
+    margin-top: 99px;
+    margin-bottom: 99px;
+
+    align-items: center;
+    text-align: center;
+}
+
+.myRow{
+    justify-content: space-between !important;
+}
+
+.Application-Image{
+    margin-top: 48px;
+    width: 610px;
+    height: 346px;
+}
+
+.text{
+    height: 381px;
+    width: 320.21px;
+    
+    h2{
+        color: #026FC2;
+        text-align: left;
+        font-family: Roboto;
```

Comment:
`font-family: Roboto;` >> `font-family: 'Roboto', sans-serif;`

The font-family property can hold several font names as a "fallback" system. If the browser does not support the first font, it tries the next font. so as a best practice don't give it less than two fonts.

fix here and all